client/base/src/main/java/io/a2a/client/ClientBuilder.java

```diff
@@ -37,7 +43,8 @@ public class ClientBuilder {
     private final AgentCard agentCard;
 
     private final List<BiConsumer<ClientEvent, AgentCard>> consumers = new ArrayList<>();
-    private @Nullable Consumer<Throwable> streamErrorHandler;
+    private @Nullable
+    Consumer<Throwable> streamErrorHandler;
```

Comment:
This looks a bit strange :-)

The original looks better, but I think 

````
@Nullable
private Consumer<Throwable> streamErrorHandler;
````
would look better. But I see we have `private @Nullable ....` 'everywhere' :-)